src/test/java/com/snowflake/kafka/connector/internal/PLConnectionServiceIT.java

```diff
@@ -0,0 +1,192 @@
+
+    private final String tableName = TestUtils.randomTableName();
+    private final String stageName = TestUtils.randomStageName();
+    private final String pipeName = TestUtils.randomPipeName();
+    private final String tableName1 = TestUtils.randomTableName();
+    private final String stageName1 = TestUtils.randomStageName();
+
+    private Credentials credentials = null;
+    
+    private Credentials getCredentials() {
+        if (credentials != null) {
+            return credentials;
+        }
+
+        AWSSecurityTokenService sts_client = AWSSecurityTokenServiceClientBuilder.standard()
+                .withEndpointConfiguration(
+                        new AwsClientBuilder.EndpointConfiguration(
+                                "sts-endpoint.amazonaws.com", "signing-region"))
+                .build();
+
+        GetSessionTokenRequest session_token_request = new GetSessionTokenRequest();
+        session_token_request.setDurationSeconds(7200);
+
+        GetSessionTokenResult session_token_result =
+                sts_client.getSessionToken(session_token_request);
+        credentials = session_token_result.getCredentials();
+        return credentials;
+    }
+
+
+    public String getId() {
+        return getCredentials().getAccessKeyId();
+    }
+
+    public String getToken() {
+        return getCredentials().getSessionToken();
+    }
+    
+    public void setConnection (boolean awsConnection) {
+        if (awsConnection) {
+            Map<String, String> properties = new HashMap<>();
+            properties.put("account", "awsaccount_pl2");
+            properties.put("user", "snowman");
+            properties.put("password", "test");
+            properties.put("role", "accountadmin");
+
+            conn = TestUtils.getConnectionService(properties);
+        } else {
+            conn = TestUtils.getConnectionService();
+        }
+    }
+    
+    @Before
+    public void before() {
+        TestUtils.executeQuery("alter session set qa_mode=true");
+        TestUtils.executeQuery("create organization AWSPL1 salesforce_id='testorg' " +
+                "organization_type=internal region_groups='PUBLIC';");
+        TestUtils.executeQuery("create or replace account awsaccount_pl2 " +
+                "with server_type=medium, license_accepted=true,internal_volume=sfc_fdndata, " +
+                "stage_volume=LOCALFS_STAGE, show_internal_params =true, change_internal_params = true, " +
+                "advanced_ui_view=true, organization=AWSPL1, type=legacy;");
+        TestUtils.executeQuery("alter account awsaccount_pl2 set qa_mode=true;");
+        TestUtils.executeQuery("alter account awsaccount_pl2 set SERVICE_LEVEL='BUSINESS_CRITICAL' " +
+                "PARAMETER_COMMENT='For PL enable';");
+        TestUtils.executeQuery("alter account awsaccount_pl2 set enforce_tls = false " +
+                "PARAMETER_COMMENT='Disable TLS check when service level is BUSINESS_CRITICAL';");
```

Comment:
can we reserve an org and account for testing purpose instead of creating for every test run?